src/jenkins.py

```diff
@@ -611,15 +611,13 @@ def has_lts_updates(proxy: state.ProxyConfig | None = None) -> bool:
     """
     try:
         current_version = get_version()
-    except JenkinsError as exc:
-        logger.error("Failed to get Jenkins version while fetching update, %s", exc)
-        raise JenkinsUpdateError("Failed to get Jenkins version.") from exc
-
-    try:
         latest_version = _get_latest_patch_version(current_version=current_version, proxy=proxy)
     except (JenkinsNetworkError, ValidationError) as exc:
         logger.error("Failed to fetch latest patch version info, %s", exc)
         raise JenkinsUpdateError("Failed to fetch latest patch version info.") from exc
+    except JenkinsError as exc:
```

Comment:
I don't see any reason for having two except blocks here. It seems a little redundant considering the exceptions you are catching already have appropriate messages

Reply:
Merged the error handlers, thank you!